src/app/helpers/filelist.helper.ts

```diff
@@ -0,0 +1,9 @@
+/**
+  * @param files Array of files to add to the FileList
+  * @returns FileList
+  */
+export function fileListItems(files: File[]): FileList {
```

Comment:
This hack aims to allow FileList editing to provide the `remove screenshot` button. Unfortunately, there is no other way to modify FileList. Input with `type=file` has attribute `files` with `FileList` type, so it would require transformation at some point anyway in case of using `File[]`

Reply:
I understand, I'm saying that it's better to refactor ix-file-input to transform to `File[]` and work with that instead of trying to piece together new `FileList`.